curriculum/challenges/english/08-coding-interview-prep/rosetta-code/long-multiplication.md

```diff
@@ -0,0 +1,81 @@
+---
+id: 5a23c84252665b21eecc7eec
+title: Long multiplication
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+Explicitly implement  <a href="https://en.wikipedia.org/wiki/long multiplication">long multiplication</a>.
+This is one possible approach to arbitrary-precision integer algebra.
+Note: In JavaScript, arithmetic operations are inaccurate with large numbers, so you will have to implement precise multiplication yourself.
```

Comment:
This line could be modified slightly and moved to the instructions section below:

```
## Instructions
<section id='instructions'>
Write a function that takes two strings of large numbers as parameters. Your function should return the product of these two large numbers as a string.
<strong>Note:</strong> In JavaScript, arithmetic operations are inaccurate with large numbers, so you will have to implement precise multiplication yourself.
</section>
```